Kitodo/src/main/java/de/sub/goobi/export/dms/DmsImportThread.java

```diff
@@ -47,8 +48,8 @@
     public DmsImportThread(Process process, String ats) {
         setDaemon(true);
         /*
-         * aus Kompatibilitätsgründen auch noch die Fehlermeldungen an alter Stelle,
-         * ansonsten lieber in neuem FehlerOrdner
+         * aus Kompatibilitätsgründen auch noch die Fehlermeldungen an alter
+         * Stelle, ansonsten lieber in neuem FehlerOrdner
```

Comment:
If you change a comment- translate it to English.

Reply:
This is just the code formatter reformatting the comment, I didn't change it.

Reply:
But if you reformat or move comment, you need to translate it.